packages/next/src/build/webpack/loaders/next-metadata-image-loader.ts

```diff
@@ -109,6 +111,16 @@ async function nextMetadataImageLoader(this: any, content: Buffer) {
               : `${imageSize.width}x${imageSize.height}`,
         }),
   }
+  if (type === 'openGraph' || type === 'twitter') {
+    const altPath = path.join(
+      path.dirname(resourcePath),
+      fileNameBase + '.alt.txt'
+    )
+
+    if (await fileExists(altPath)) {
+      imageData.alt = await fs.readFile(altPath, 'utf8')
+    }
```

Comment:
Maybe more of a question for @sokra - possible to have this added as a dependency, and is there a way to cache it?